Dockerfile

```diff
@@ -24,8 +24,8 @@ RUN apt-get update && apt-get install -y --allow-unauthenticated --no-install-re
 		python3-pip \
 		python3-dev \
 		git \
-		r-base-core=4.3.3-1.2004.0 \
-		r-base-dev=4.3.3-1.2004.0 \
+		r-base-core=4.3.3-1.2204.0 \
+		r-base-dev=4.3.3-1.2204.0 \
```

Comment:
Just out of curiosity, how did you decide the version here?

Reply:
It's by the name of the ubuntu version we use for the dockerfile. Previously it was ubuntu focal which was 20.04, but now it's ubuntu jammy so it's 22.04.